lib/jsdoc.js

```diff
@@ -167,14 +168,41 @@ function parseParamKeyword(text) {
 }
 
 function parseReturnKeyword(text) {
-    const output = { type: DEFAULT_TYPE, desc: '' };
+    const output = {
+        doctype: {

```

Comment:
Ok, I see that we have some old code with non consistent data structure. However, I'm not sure that it is a good idea to introduce the new structure of output data. 
I prefer to keep clean and consistent API and release this fix with new major version. What do you think about that?

Let's keep output in the following format: 
```
{
    type: { ... },
    description: '...'
}
```

Reply:
Yes definitely, a major release would be the best option. I did not want to break anything so I introduced new keys in the current format. But if you're willing to do a major release, the old format of `method.return`:

```typescript
interface SvelteMethodReturnItem {
    type: string|string[];
    desc: string;
}
```
would become this:

```typescript
interface SvelteMethodReturnItem {
    type: JSDocType;
    description: string;
}